src/main/scala/net/akmorrow13/endive/utils/Window.scala

```diff
@@ -27,7 +27,7 @@ class Window(tf: String,
                   region: ReferenceRegion,
```

Comment:
As in merge my pr and then put this on top of it or don't include this as part of your PR it seems unnecessary/tangential to the rest of the thing.